ComponentKit/Accessibility/CKComponentAccessibility.mm

```diff
@@ -24,7 +24,11 @@ static CKViewComponentAttributeValueMap ViewAttributesFromAccessibilityContext(c
     accessibilityAttributes[@selector(setAccessibilityIdentifier:)] = accessibilityContext.accessibilityIdentifier;
   }
   if (accessibilityContext.isAccessibilityElement) {
+#if TARGET_OS_IPHONE
     accessibilityAttributes[@selector(setIsAccessibilityElement:)] = accessibilityContext.isAccessibilityElement;
+#else
```

Comment:
That's reasonable. Apple's decision to make iOS also `TARGET_OS_MAC` defines the order, but I can make the else simply `TARGET_OS_MAC`.